public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -168,6 +188,11 @@ export class PanelChrome extends Component<Props, State> {
         }
       }
     }
+
+    // The timer depends on panel width
+    if (width !== prevProps.width) {
```

Comment:
will be quite a bit off for panels with right legends (non full width charts) via vizlayout, but i dont have any concrete ideas/solutions.